### PR TITLE
Fixes environment configuration

### PR DESCRIPTION
**Pull request by b00kdev**
Hi. I noticed that my environment config was being ignored when trying to use Blockfrost and the Cardano testnet. It looks like `load.rs` wasn't correctly pulling the values from the environment and updating the `TomlValue`. This PR fixes that (though I'm fairly new to Rust, so I'm not sure if this is the best way).

### CODE REVIEW

File: src/load.rs [hunk 1]
```diff
@@ -59,20 +59,28 @@ use crate::Error;
...
-    let mut toml_value = match &config_file {
+    let toml_value = match &config_file {
         Some(file_path) => load_toml_from_path(file_path)?,
         None => TomlValue::Table(Default::default()),
     };
 
-    if let Ok(var) = env::var("BLOCKFROST_NETWORK_ADDRESS") {
-        toml_value["blockfrost-network-address"] = TomlValue::String(var);
-    }
+    Ok(toml_value.as_table().map_or(toml_value.clone(), |x| {
```
Comment (jpochyla) [inline src/load.rs:67]: Hi @b00kdev! I'll review the code. It seems that the essential fix is:
- using the correct env vars,
- using the correct config keys.

Do you think we could keep the original style of overriding the values? Has there been any problem with keeping the `mut` value and `[]` indexing? From what I've seen, `[]` should automatically work on `toml::Value`s that are tables.

File: src/load.rs [hunk 2]
```diff
@@ -59,20 +59,28 @@ use crate::Error;
...
+            toml_table.insert("cardano_network".to_string(), TomlValue::String(var));
+        }
+
+        if let Ok(var) = env::var("BLOCKFROST_IPFS_NETWORK") {
+            toml_table.insert("ipfs_network".to_string(), TomlValue::String(var));
+        }
 
-    Ok(toml_value)
+        TomlValue::Table(toml_table)
+    }))
```
Comment (jpochyla) [inline src/load.rs:83]: Something like this (untested):

```suggestion
    let mut toml_value = match &config_file {
        Some(file_path) => load_toml_from_path(file_path)?,
        None => TomlValue::Table(Default::default()),
    };

        if let Ok(var) = env::var("BLOCKFROST_PROJECT_ID") {
            toml_value["project_id"] = TomlValue::String(var);
        }

        if let Ok(var) = env::var("BLOCKFROST_CARDANO_NETWORK") {
            toml_value["cardano_network"] = TomlValue::String(var);
        }

        if let Ok(var) = env::var("BLOCKFROST_IPFS_NETWORK") {
            toml_value["ipfs_network"] = TomlValue::String(var);
        }
        
        Ok(toml_value)
```

**Comment by mmahut** [comment]
ping @b00kdev :) do you wish to continue with this PR?

**Comment by b00kdev** [comment]
@mmahut @jpochyla Hello, hello! I am sorry, I somehow missed the notifications that @jpochlya had reviewed the code. I think the review requests make perfect sense and would be happy to see those added. I can make the change and update the PR, if that works. Thank you for the ping!

**Pull request closed without merging**